docs/versioned-plugins/include/attributes-ls-vpr.asciidoc

```diff
@@ -0,0 +1,4 @@
+// These VPR settings correspond to the LSR settings in logstash/docs/include/attributes-ls.asciidoc
+// When we update one, we must update the other
+
+:ecs-default: Any value in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] for pipeline.ecs_compatibility other than disabled is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
```

Comment:
Same as noted in upstream [PR comment](https://github.com/elastic/logstash/pull/13083#discussion_r717034110).